core/src/main/java/com/graphhopper/util/shapes/Polygon.java

```diff
@@ -192,4 +224,26 @@ public double getMaxLat() {
     public double getMaxLon() {
         return maxLon;
     }
+
+    @Override
+    public String toString() {
+        return "polygon (" + lat.length + " points)";
+    }
+
+    public static Polygon parsePoints(String pointsStr, double growFactor) {
+        String[] arr = pointsStr.split(",");
+
+        double[] lats = new double[arr.length /2];
```

Comment:
@andreylh we need a small length. E.g. if arr.length%2==1 throw IllegalArgumentException("incorrect polygon specified")